locallib.php

```diff
@@ -1656,6 +1656,24 @@ function bigbluebuttonbn_get_recording_data_row_type($recording, $bbbsession, $p
     return $OUTPUT->action_link('#', $text, null, $linkattributes) . '&#32;';
 }
 
+/**
+ * Helper function to handle yet unknown recording types
+ *
+ * @param string $playbacktype : for now presentation, video, statistics, capture, notes, podcast
+ *
+ * @return either the matching language string or a capitalised version of the provided string
+ */
+function bigbluebuttonbn_get_recording_type_text($playbacktype) {
+    // Check first if string exists
```

Comment:
If you run the codechecker you should see an error here. Comments must end in a dot .

Reply:
Ok, that's right. Just amended and shortened the comment.